lib/ui/form/form_screen.dart

```diff
@@ -37,6 +42,12 @@ class FormScreen extends ConsumerStatefulWidget {
 }
 
 class FormScreenState extends ConsumerState<FormScreen> {
+  var _showStartSurveyButton = true;
+  var _showNextSurveyButton = false;
+  var _showSubmitSurveyButton = false;
+  var _showCloseButton = false;
```

Comment:
Hmmm, makes me wonder. 

Should I use state providers instead of this approach? 💭 

cc @luongvo @Shayokh144